src/framework/preferences/js/LetterSpaceEnactor.js

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2017-2018 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_3_0_0 = fluid_3_0_0 || {};
+
+(function ($, fluid) {
+    "use strict";
+
+    /*******************************************************************************
+     * letterSpace
+     *
+     * Sets the letter space on the container to the number of units to increase
+     * the letter space by. If a negative number is provided, the space between
+     * characters will decrease. Setting the value to 1, unit to 0,  will use the
```

Comment:
"Setting the value to 1, unit to 0" -> "Setting the value to 1 or unit to 0". Is it missing "or", which makes it sound like both model paths need to be set.